v2/emailpassword/serverless/with-aws-lambda/setup-lambda.mdx

```diff
@@ -1,74 +1,42 @@
 ---
```

Comment:
I see that you have renamed this file. Please add the older file back with a `<Redirector ...>` component which redirects the user to the right page. You do not need to inlcude this in the sidebar.json. This is only done cause google might have linked to this file and then we don't want to give a 404 to the user.

Reply:
done